tests/cmd/stability/main.go

```diff
@@ -100,6 +100,46 @@ func main() {
 		},
 		Monitor:          true,
 		BlockWriteConfig: conf.BlockWriter,
+		SubValues: `pd:
+  affinity:
```

Comment:
I recommend abstract the template and render the `namespaces` field, maintaining similar yaml literal in 4 places is error-prone